setup.py

```diff
@@ -93,12 +98,34 @@ def run(self):
             )
 
 
+def get_version(version: "ScmVersion") -> str:
+    from setuptools_scm import version as scm_version
```

Comment:
We normally avoid imports inside methods. You have a good reason to do this, but can you add a comment to explain why?